ioctl/newcmd/action/actiondeposit.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+)
+
+// Multi-language support
+var (
+	_depositCmdShorts = map[config.Language]string{
+		config.English: "Deposit rewards to rewarding fund",
+		config.Chinese: "将奖励存入奖励基金",
+	}
+	_depositCmdUses = map[config.Language]string{
+		config.English: "deposit AMOUNT_IOTX [DATA] [-s SIGNER] [-n NONCE] [-l GAS_LIMIT] [-p GAS_PRICE] [-P PASSWORD] [-y]",
+		config.Chinese: "deposit IOTX数量 [数据] [-s 签署人] [-n NONCE] [-l GAS限制] [-p GAS价格] [-P" +
+			" 密码] [-y]",
```

Comment:
make in one line

Reply:
I'll change `actiontransfer`, `stake2withdraw` and `xrctransferfrom` accrodingly, but I'll keep the lines split if it's too long.